src/Shared/KnownConfigNames.cs

```diff
@@ -5,8 +5,10 @@ namespace Aspire.Hosting;
 
 internal static class KnownConfigNames
```

Comment:
Unrelated to this PR, but this class name seems misleading. These are env var names, not config keys.

Reply:
Debatable :) They are perfectly valid config keys. All environment variable names are valid config key names.

Reply:
I agree with Drew. And this file should be specific to hosting. Dashboard [has its own file](https://github.com/dotnet/aspire/blob/f0494ca5691ce279180fb284df32e582bf335834/src/Shared/DashboardConfigNames.cs). Host has it. If some are duplicated then that's fine. The two apps just happen to have some config settings with the same name.

Reply:
I was talking about them being env var names ;)